phpBB/styles/prosilver/theme/common.css

```diff
@@ -603,6 +604,53 @@ a.header-avatar span:after {
 	display: none !important;
 }
 
+/* Smiley Dropdown---------------------------------------- */
+.smiley-box {
+	float: right;
+	width: 100%;
+}
+
+.smiley-box .smiley-image {
+	margin: 3px;
+}
+
+.dropdown .smiley-content {
+	width: 200px;
+}
+
```

Comment:
why is this chained with dropdown? if needed use the `smiley-drowndown` class to avoid mixing components